src/templates/staff/assignment.html

```diff
@@ -219,12 +226,201 @@
 </nav>
 <div class="container-fluid">
     <div class="row">
-        <div class="col-md-6 col-lg-8 col-xl-9">
+        {% if is_admin %}
+            <div class="card" style="flex-direction: row; flex: 1;">
+                <div class="card-body">
+                    <div class="container-fluid" style="padding: 0;">
+                        <div class="row">
+                            <div class="col-md-6 col-lg-6 col-xl-4" style="align-self: center;">
```

Comment:
Let's add a `text-center` class (which is bootstrap for `text-align: center;` attribute) to this div so that the title is centered when the window is small. I think it will also look good if we add some padding on the bottom. If we add `pb-3 pb-md-0` class, the padding on bottom would be 3 on small screen and 0 on larger screen.